scout/models/case/case_loading_models.py

```diff
@@ -221,6 +221,7 @@ class SampleLoader(BaseModel):
     rhocall_wig: Optional[str] = None
     rna_alignment_path: Optional[str] = None
     rna_coverage_bigwig: Optional[str] = None
+    rna_sample_id: Optional[str] = None
```

Comment:
I would not add this key right now in this patch PR. I'd wait and think about it a bit instead. Could as well go in the next minor no?